### ISSUE TITLE
[bug report] Seed must be between 0 and 2**32 - 1; gym version too old

### ISSUE THREAD

**Issue by ncble** [araffin/robotics-rl-srl#42]
**Describe the bug**
- [1.] ValueError: Seed must be between 0 and 2**32 - 1
- [2.] ImportError: cannot import name 'prng' (for gym>=0.11.0)

**Code example**
The following code can reproduce the problem with the current version of origin/master:
python -m environments.dataset_generator --seed=1 --name=debug

**System Info**
gym version == 0.10.3 

## Solution

- [1.] Either add line seed = seed % 2^32 or comment out the line prin.seed(seed)
- [2.] see the [link](http://rl-tau-2019.wikidot.com/forum/t-10724076/issues-with-from-gym-spaces-import-prng): downgrade gym to gym==0.10.3

**Comment by araffin** [comment]
Hello,
We would appreciate a PR that solves this issue ;)

For a possible fix, see: https://github.com/hill-a/stable-baselines/blob/master/stable_baselines/common/misc_util.py#L93

**Comment by ncble** [comment]
Yes, the PR is coming soon (within one or two days). I would solve the other problem at the same time! :)
